packages/task/src/browser/task-service.ts

```diff
@@ -516,9 +528,7 @@ export class TaskService implements TaskConfigurationClient {
             this.messageService.error(error.message);
             return undefined;
         }
-        return this.runTasksGraph(task, tasks, {
-            customization: { ...customizationObject, ...{ problemMatcher: resolvedMatchers } }
-        }).catch(error => {
+        return this.runTasksGraph(task, tasks, option).catch(error => {
             console.log(error.message);
```

Comment:
I wonder why do we not log full stack trace here and why we don't use `console.error`?  The same above at `this.logger.error(error.message)`.

Reply:
Fixed, please take a look https://github.com/eclipse-theia/theia/pull/7696/commits/a5367f9314321de98ccf3894b4aaea4d2652d91e